source/sonata_simulation.rst

```diff
@@ -96,6 +97,27 @@ example::
        "integration_method" : 2
   }
 
+Format of the electrodes_file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The format of the weights file is described below:
+
+.. table::
+
+  ============================= ======================= ========== =============== ============= ==========================================
+  Group                         Field                   Type       Shape           Requirement   Description
+  ============================= ======================= ========== =============== ============= ==========================================
+  /electrodes/{electrodename}   layer                   utf8       1               Optional      Layer of the circuit in which {electrodename} is located. If the electrode is in a region without cortical layers, then "NA". If the electrode is outside the brain, then "Outside"
+  /electrodes/{electrodename}   position                float32    3               Mandatory      Position of {electrodename} in microns, in cartesian coordinates
+  /electrodes/{electrodename}   region                  utf8       1               Optional      Region in which {electrodename} is located
+  /electrodes/{electrodename}   type                    utf8       1               Optional      Either EEG or LFP
+  /electrodes/{electrodename}   offset                  uint64     1               Optional      Column index of the coefficients for {electrodename} in /electrodes/{population_name}/{node_id}
+  /electrodes/{population_name} {node_id}               float64    N_seg x N_elec  Mandatory     Scaling factor for each segment in the corresponding neuron, in mV/nA
```

Comment:
What happens if an `{electrodename}` is the same as a `{population_name}`? Or, to phrase it a different way, how can one distinguish `{electrodename}` from `{population_name}` in the `/electrodes` namespace?